src/NewChain/AuxiliaryChain.ts

```diff
@@ -23,6 +23,7 @@ export default class AuxiliaryChain {
   private chainDir: string;
   private sealer: string;
   private deployer: string;
+  private maxRetries = 5;
```

Comment:
This class is called `AuxiliaryChain`. The name `maxRetries` is not clear in that context. A better name could be something like `private timeToWaitForAccountsToUnlockInSeconds = 30`. This way:

1. it is clear that the total wait time will be max 30 seconds. The total wait time is otherwise totally unclear from the number of tries
2. it is clear what the variable is used for and that the time is measured in seconds (not e.g. milliseconds).

You would then hard-code the number of tries in the code below, e.g. 10, and divide the max wait time by the number of tries to get the wait time. Ping me if this is not explained very clearly and I can expand 😅